ckanext/nextgeoss/helpers.py

```diff
@@ -449,9 +453,16 @@ def collection_information(collection_id=None):
 
 
 def get_extras_value(extras, extras_key):
+    '''
+        Return the value from extras from a given key.
+    '''
+    value = ''
```

Comment:
I can see the rationale behind this but as far as we know some of the extra values may be integers or other types. For those cases it wouldn't be appropriate to set their default to an empty string.

So I think it would be best to leave the default for `value` as `None` and set the default for `collection_name` in the template.

Reply:
We only have strings

Reply:
Do as you deem best.